en/tutorials/linux_sbus.md

```diff
@@ -29,7 +29,7 @@ GND ->S.bus gnd
 ![](https://github.com/crossa/raspx4-sbus-rc-in/blob/master/sbusToSerial_small.jpg?raw=true)  
 
 ## Example
-![](https://raw.githubusercontent.com/crossa/raspx4-sbus-rc-in/master/example.jpeg)
+![](https://raw.githubusercontent.com/crossa/raspx4-sbus-rc-in/master/example.png)
```

Comment:
Although not perfect, seems acceptable to me. Thanks for the contribution @crossa